python/glow/gwas/lin_reg.py

```diff
@@ -23,11 +23,13 @@ def linear_regression(genotype_df: DataFrame,
                       add_intercept: bool = True,
                       values_column: Union[str, Column] = 'values',
                       dt: type = np.float64,
-                      verbose_output: bool = False) -> DataFrame:
+                      verbose_output: bool = False,
+                      intersect_samples: bool = False,
```

Comment:
Do you think we should set `intersect_samples: bool = True` by default?

Reply:
actually let's keep it false, and I will make a quickstart guide that demos this functionality, showing confounded results vs non-confounded (with `intersect_samples` set to True)